### ISSUE TITLE
Whenever I go into localhost:9000 it's just a blank screen, nothing appears

### ISSUE THREAD

**Issue by FlexibleYT** [DestinyTrialsReport/DestinyTrialsReport#138]
?

**Comment by v-fedorov** [comment]
Are you getting any errors in the console? Make sure to use the develop branch if you're not already

**Comment by FlexibleYT** [comment]
I wasn't using the developer branch because everytime I do that, I get a few errors ( I am new to javascript so this is a new thing for me that I want to develop on. I only do php and css + html)

And yes, I do get errors and here are some screenshots:
http://prntscr.com/g0mic8
this bower thing used to work before, but even when it worked... I got this error when trying to node manifest.js

But if I just grunt serve even though I get errors, I get this: 
http://prntscr.com/g0miqg

Hope you can help ;) sorry for all of this if I am wasting your time, but when I get the server to work... Oh damn do I have plans.

**Comment by v-fedorov** [comment]
the readme is outdated. you dont need to run the node manifest.js command anymore.

in the develop branch, run `npm install` and then `bower install`
after that `grunt serve` should work

**Comment by FlexibleYT** [comment]
Yes but whenever I run bower install it says "bower ENORESOLVER   Bower resolver not found: bower-npm-resolver"
"

**Comment by FlexibleYT** [comment]
Nevermind, found a solution 

> npm install -g bower-npm-resolver

**Comment by v-fedorov** [comment]
also try running `npm install -g bower`

**Comment by FlexibleYT** [comment]
How do I give someone a badge?

**Comment by v-fedorov** [comment]
badges are populated here

https://github.com/DestinyTrialsReport/DestinyTrialsReport/blob/develop/app/components/player/overview.partial.html#L11

what exactly are you trying to do?

**Comment by FlexibleYT** [comment]
Develop. I am gonna take badges to a new level lol, I need to know how to give it so I can preview it. I'm redesigning, adding new features etc...

**Comment by v-fedorov** [comment]
oh jeez,
heres the html for a basic badge

```
<span class="player-hazard player-hazard--donator"
     data-template-url="shared/popovers/charity.popover.html"
     data-placement="bottom" bs-popover>
Badge Name
</span>
```

**Comment by FlexibleYT** [comment]
But how can you give a certain psn or gamertag a badge? For eg. Donator badge

**Comment by FlexibleYT** [comment]
Sorry for bein so annoying, ^^ was my last question

**Comment by v-fedorov** [comment]
that information is stored in our database and is accessed via our API.
Everything in this repo is clientside only

**Comment by FlexibleYT** [comment]
Aha ok. Thanks

**Comment by FlexibleYT** [comment]
Is your api accessible?

**Comment by v-fedorov** [comment]
Yeah but you're not going to able to push any data into it

**Comment by FlexibleYT** [comment]
Oh ok. Nvm

**Comment by v-fedorov** [comment]
You think we would just allow anyone to give themselves badges like that? Lol

**Comment by SunburnedGoose** [comment]
Need that local developer DB instance.

**Comment by FlexibleYT** [comment]
Local bro lol
